fetch.bs

```diff
@@ -8919,6 +8919,10 @@ set; otherwise false.
 <p>The <dfn attribute for=Request><code>signal</code></dfn> getter steps are to return <a>this</a>'s
 <a for="Request">signal</a>.
 
+<p class="note"><a>this</a>'s <a for="Request">signal</a> is always initialized in the
+<a href="#ref-for-request-signal②">constructor</a> and when
+<a href="#ref-for-request-signal⑤>cloning</a>.
```

Comment:
Let's capitalize "This" (or reword). We also can't use those backreferences unfortunately as they are not stable. Although maybe we could add these as ID to those places to make them stable. (Although it might well be that this ends up renumbering others in a weird way, not sure.)

Reply:
Added identifiers to the list items and updated links. They all work now.